tuf-spec.md

```diff
@@ -1052,19 +1046,43 @@ discuss next.
     directory separator and does not start with a directory separator, akin to
     <a>TARGETPATH</a>.
 
+  : <dfn>TERMINATING</dfn>
+  ::
+    A boolean indicating whether subsequent delegations should be considered.
+
+    As explained in the [Diplomat paper
+    ](https://theupdateframework.io/papers/protect-community-repositories-nsdi2016.pdf),
+    terminating delegations instruct the client not to consider future trust
+    statements that match the delegation's pattern, which stops the delegation
+    processing once this delegation (and its descendants) have been processed.
+    A terminating delegation for a package causes any further statements about a
+    package that are not made by the delegated party or its descendants to be
+    ignored.
```

Comment:
Git Nit Pick: I don't fully understand why this block is in this diff. It was introduced in 0713382db748be0c9d57a374cd9e695010cf2fa2 and did not [appear in the old draft diff](https://github.com/theupdateframework/specification/compare/fd85a8a...draft#diff-b0020aab85257910a5a31f248c201a81cb35e1939545c4b45515e9049e78cdffR885-R886). Maybe a rebase hiccup? I guess we can live with it.

Reply:
It's in the HEAD of the current draft branch: https://github.com/theupdateframework/specification/blame/draft/tuf-spec.md#L888 and appears below the fold (click the expand down button) in the diff you linked to.

Reply:
Oh, I understand now what happened. It's in the diff because you shuffled the section a bit during rebase, right? the paragraph that describes `TERMINATING` used to be further above. I suppose to align with the order in the json sample? Either way, it's all good, and given that you have already rebase, we can close here. Thanks again!